src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/Commons.cs

```diff
@@ -193,6 +198,11 @@ public enum HWType
         /// </summary>
         CPU = 0x1000,
         /// <summary>
+        /// SIMD in CPU in possible.
```

Comment:
It is spelling mistake, but I don't know what should be at the end. Perhaps `if possible`.